src/index.jsx

```diff
@@ -25,6 +26,7 @@ import {ImageEditViewContainer} from './containers/ImageEditView';
 
 
 const reducers = {
+  app: appReducer,
```

Comment:
is it unnecessary to spread the state across different stores, eg should this "appState" be combined with the modelState?